css/style.css

```diff
@@ -0,0 +1,418 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter&family=Roboto:wght@100&display=swap');
+
+body {
+  margin: 0;
+  overflow-x: hidden;
+  padding: 0;
+}
+
+.container {
+  width: 100%;
+  height: 100vh;
+}
+
+header {
+  padding: 15px 24px;
+  font-family: 'Arial, Helvetica', sans-serif;
+  justify-content: space-between;
+  display: flex;
+  color: rgb(255, 255, 255);
+  box-sizing: border-box;
+  width: 100%;
+  background: #3c3a39;
+}
+
+header .logo {
+  color: #fff;
+  text-decoration: none;
+}
+
+.fa-bars::before {
+  color: #fff;
+  text-decoration: none;
+}
```

Comment:
You should decrease your styling (e.g header .logo and .fa-bars::before ).
they have the same styling if possible you can include them in one class.